packages/framework-types/src/config.ts

```diff
@@ -241,15 +244,20 @@ export class BoosterConfig {
       }
     }
   }
+
+  // TTL for events stored in dispatched events table. Default to 5 minutes (i.e., 300 seconds).
+  public dispatchedEventsTtl = 300
```

Comment:
Do you mind placing this along with the other properties, before all the methods? I will be easier to find there as it is together with the other props